promtimer/promtimer.py

```diff
@@ -312,9 +312,15 @@ def main():
 
     cbcollects = get_cbcollect_dirs()
     if len(cbcollects) == 0:
-        logging.error('No "collectinfo*.zip" files or "cbcollect_info*" '
-                      'directories found')
-        sys.exit(1)
+        if os.path.isdir(STATS_SNAPSHOT_DIR_NAME):
+            # Found stats directory, assume we're inside an unzip'd cbcollect
+            # directory
+            cbcollects.append(".")
```

Comment:
It's a little bit strange that the data source that shows in Grafana is "." but I guess it works. We could think about coming up with a better name than ".". Perhaps we could find the name of the directory in which the cbcollect resides. But this is simple and it works and for now it's good enough. Maybe we can improve it later.